loop/config.js

```diff
@@ -529,6 +529,17 @@ var conf = convict({
       default: "",
       env: "ROOMS_HKDF_SECRET"
     }
+  },
+  ga: {
+    activated: {
+      doc: "Should we send POST /events data to Google Analytics while true.",
+      default: false,
+      format: Boolean
```

Comment:
Since the default is false, it is unlikely that we will use ``activated: false`` but I will double check that because in that case we might have the same problem with all other boolean config parameters.